docs/src/design/chips/flow-ctrl.md

```diff
@@ -13,16 +13,12 @@ Branch chip uses columns to record the following information.
   - Implements 32-bit address validation through dedicated range-checking components(`next_pc_range_checker, target_pc_range_checker, next_next_pc_range_checker`).
 - ​Operand Handling System​​
   - Stores three register/immediate values following MIPS three-operand convention (`op_a_value, op_b_value, op_c_value`).
-  - Contains special flag detection for zero-register operand scenarios (`op_a_0`).
-- ​​Instruction Semantics Encoding​​
-
-  Embeds five mutually exclusive flags corresponding to MIPS branch opcodes (`is_beq, is_bltz, is_blez, is_bgtz, is_bgez`).
+- ​​Instruction Semantics Encoding
+  - Embeds five mutually exclusive flags corresponding to MIPS branch opcodes (`is_beq, is_bltz, is_blez, is_bgtz, is_bgez`).
 - ​Execution State Tracking​​
-
-  Maintains dual execution path indicators for taken/not-taken branch conditions(`is_branching, not_branching`). 
+  -  Maintains dual execution path indicators for taken/not-taken branch conditions(`is_branching, not_branching`). 
 - ​Comparison Logic Core​​
-
-  Evaluates signed integer relationships between primary operands, generating equality, greater-than, and less-than condition flags (`a_eq_b, a_gt_b, a_lt_b`). 
+  - Evaluates signed integer relationships between primary operands, generating equality, greater-than, and less-than condition flags (`a_eq_b, a_gt_b, a_lt_b`). 
```

Comment:
The beginning item is not alligned. Check if it will make the format not correct